### PR TITLE
Log4j Configuration

### PR DESCRIPTION
**Pull request by MunifNagi**
Adding logger

### CODE REVIEW

File: src/classes/Employee.java [hunk 1]
```diff
@@ -32,7 +32,9 @@ public Flight chooseFlight() {
         System.out.println("Please enter the Flight you wish to Book.");
 
         Scanner keyboard = new Scanner(System.in);
-        return flights.get(Integer.parseInt(keyboard.nextLine()) - 1);
+        int chosen=(Integer.parseInt(keyboard.nextLine()));
+        if (chosen>flights.size()){return  null;}
```
Comment (VbKoshak) [inline src/classes/Employee.java]: either 
```
if (chosen > flights.size()) {
...
}
```
or 
```
if (chosen > flights.size()) return null;
```
But the first one is more preferable

File: src/classes/Main.java [hunk 2]
```diff
@@ -21,8 +31,10 @@ public static void main(String[] args) {
         user1.greeting();
         user2.greeting();
         Flight chosen = user1.chooseFlight();
+        if (chosen==null){logger.warn("No Flight has been chosen");}
```
Comment (VbKoshak) [inline src/classes/Main.java]: format here

File: src/classes/Main.java [hunk 3]
```diff
@@ -21,8 +31,10 @@ public static void main(String[] args) {
...
         user1.bookTicket(chosen, user2);
         user2.getTickets();
+        logger.info("Exiting application.");
```
Comment (VbKoshak) [inline src/classes/Main.java]: this empty line is not necessary

**Review by VbKoshak — CHANGES_REQUESTED** [review]

**Review by VbKoshak — APPROVED** [review]
lgtm

**Comment by MunifNagi** [comment]
outdated pull request

**Pull request closed without merging**